build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -611,8 +611,11 @@ def iree_generated_trace_runner_test(self,
       drivers = [it[1] for it in target_backends_and_drivers]
 
     name_block = _convert_string_arg_block("NAME", name, quote=False)
+    # For now we assume that the generator target is a py_binary with a single
+    # source .py file named like it.
+    generator_py = re.sub(':(\\w+)', '\\1.py', generator)
```

Comment:
Regex seems like overkill here.


```suggestion
    generator_py = f"{generator.split(':')[-1]}.py"
```

Reply:
nice!